### ISSUE TITLE
Map Details and Player Scale

### ISSUE THREAD

**Issue by SpineyPete** [jdolan/quetoo-data#125]
This is something that's a minor issue with some maps, but the scale of certain textures and brushes is a bit off in some maps. This is a classic problem but easy to fix in most cases. Just run around a map in third person and look if the player looks like a midget. For example on Torn the individual stairs should probably be half as high, there's also the barrels which are huge. Quetoo is not a realistic game so there's no need for real world scale, but it should be "roughly" right imo.

**Comment by jdolan** [comment]
I agree. Most of our maps use 8u stairs now, and Torn should be updated as such. I haven't tried this already because of the light sources on the stair risers. But I'm sure there's a workable solution.

This was / is even worse in Quake2, btw. This is a big part of why I experimented with scaling our player model and bounding box up from Quake2's comically small size. Stairs are up to grunt's waist in stock Quake2 😂 The challenge, tho, is that a lot of maps have major movement issues if you increase the player size to a more realistic value in relation to 16u stairs. But you'll see in the code PM_MINS, PM_MAXS, and PM_SCALE (currently 1.f).

**Comment by jdolan** [comment]
@skies912 fixed in 0a4a62c8113fb63931a88b2fb7632536b0de1a41 and 90a0e611dc4ff8b47ef07346b768fbacffff8f8f
